ports/esp8266/network_wlan.c

```diff
@@ -519,6 +519,9 @@ STATIC const mp_rom_map_elem_t wlan_if_locals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_PM_NONE), MP_ROM_INT(NONE_SLEEP_T) },
     { MP_ROM_QSTR(MP_QSTR_PM_PERFORMANCE), MP_ROM_INT(MODEM_SLEEP_T) },
     { MP_ROM_QSTR(MP_QSTR_PM_POWERSAVE), MP_ROM_INT(LIGHT_SLEEP_T) },
+    #ifdef MICROPY_PY_NETWORK_MODULE_GLOBALS_INCLUDEFILE
+    #include MICROPY_PY_NETWORK_MODULE_GLOBALS_INCLUDEFILE
+    #endif
```

Comment:
Please add the `IF_xxx` and `SEC_xxx` constants explicitly, not by including this header.